dacite/config.py

```diff
@@ -0,0 +1,103 @@
+from dataclasses import dataclass, field as dc_field, fields, Field
+from typing import Dict, Any, Callable, List, Optional, Type
+
+from dacite.data import Data
+from dacite.dataclasses import has_field_default_value
+from dacite.exceptions import InvalidConfigurationError
+from dacite.types import cast_value
+
+
+class ValueNotFoundError(Exception):
```

Comment:
Why this exception is not in `dacite.exceptions`?

Reply:
Also, I would use your own class a base to make it easier to catch `dacite` exceptions.

Reply:
It is intended. This is internal exception of `config` module - he will be caught by `core.from_dict` function. In `dacite.exceptions` I want to have only "user-level" exceptions.

Does it make sense?

Reply:
What if you forget to catch internal exceptions somewhere. I propose to move all exceptions and to create `InternalDaciteError` for special cases.

Reply:
Hmm, ok, I will think about it, but I like that current solution is less coupled with rest of code - this module is more independent.